No-op patch to placate the build system

This is a placeholder in case we need to patch the sparse caster in the
future.

--- include/nanobind/eigen/sparse.h
+++ include/nanobind/eigen/sparse.h
@@ -1,1 +1,1 @@
-/*
+/*
